.setup/install_submitty/install_site.sh

```diff
@@ -408,15 +408,15 @@ chmod 540 ${SUBMITTY_INSTALL_DIR}/site/cgi-bin/*
 chmod 550 ${SUBMITTY_INSTALL_DIR}/site/cgi-bin/git-http-backend
 
 mkdir -p "${NODE_FOLDER}/.vue-global-types"
-chgrp "${PHP_USER}" "${NODE_FOLDER}/.vue-global-types"
+chown -R ${PHP_USER}:${PHP_USER} "${NODE_FOLDER}/.vue-global-types"
```

Comment:
please add the double quotes around the users, it's defensive programming in case the variable doesn't exist/is empty string.